### PR TITLE
Doc fixes: gettings-started/runtime -- typo

### PR DESCRIPTION
**Pull request by tomgs**
I'm going through the docs and this is the second typo I found. Please don't merge yet, I might find a few more:) I'll write here once I'm done so you guys can merge. 

Also, that's an awesome thing to complain about - thanks for being responsive (and for writing the crate!).

### CODE REVIEW

**Comment by tomgs** [comment]
Looks fine for now. Please merge:)

File: content/docs/getting-started/runtime.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ runtime or no work will be performed.
 
 One of the unique aspects of Tokio is that futures can be spawned on the runtime
 from within other async tasks. Tasks are the application’s “unit of logic”.
-They are similar to [Go’s goroutine] and [Erlang’s process], but asynchronous.
+They are similar to [Go’s goroutine](https://gobyexample.com/goroutines) and [Erlang’s process](http://erlang.org/doc/reference_manual/processes.html), but asynchronous.
```
Comment (Ralith) [inline content/docs/getting-started/runtime.md]: I think these should be defined at the bottom of the file like other links for consistency.

File: content/docs/getting-started/futures.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ finds the function associated with it and calls it immediately. This is a
 
 The rust asynchronous model uses a **pull** based model. Instead of a `Future`
 being responsible for pushing the data into a callback, it relies on **something
-else** asking it it is complete or not. In the case of Tokio, that **something
+else** asking it if it is complete or not. In the case of Tokio, that **something
```
Comment (Ralith) [inline content/docs/getting-started/futures.md]:
```suggestion
else** asking if it is complete or not. In the case of Tokio, that **something
```

Comment (ultrasaurus) [reply content/docs/getting-started/futures.md]: this seems like a nice suggestion

File: content/docs/getting-started/runtime.md (hunk 1 above)
Comment (ultrasaurus) [reply content/docs/getting-started/runtime.md]: also this overlaps with github.com/tokio-rs/website/pull/391 -- would be great if someone made an isolated PR that just fixed these links so we can avoid merge craziness!

**Comment by ultrasaurus** [comment]
thanks @tomgs for these contributions!  can you take a look at requested changes and, if you merge master, it'll now generate a nice preview.

File: content/docs/getting-started/futures.md (hunk 2 above)
Comment (tomgs) [reply content/docs/getting-started/futures.md]: Ah, yes, it really does! Comitting.

**Comment by tomgs** [comment]
Hey @ultrasaurus, fixing now.

File: content/docs/getting-started/runtime.md (hunk 1 above)
Comment (tomgs) [reply content/docs/getting-started/runtime.md]: Hey @ultrasaurus, I can probably just remove my link changes from here as they are fixed by the other PR.

**Comment by tomgs** [comment]
@ultrasaurus So, basically, after cleaning up and removing merge conflicts, this is a single word PR! :) No problem in merging I presume.

**Pull request merged**